docs/tutorial/rust/src/getting_started.md

```diff
@@ -12,15 +12,10 @@ cargo new memory
 cd memory
 ```
 
-Then we edit `Cargo.toml` to add the slint dependency:
+Then we edit `Cargo.toml` to add the slint dependency using `cargo add`:
 
-```toml
-[package]
-#...
-edition = "2021"
-
-[dependencies]
-slint = "0.3.4"
+```sh
+cargo add slint
```

Comment:
Perhaps this should be a variant of `cargo add` that specifies the version. Since the entire tutorial is also versioned, and so is the C++ one (by release branch).